No Android build being released

Maybe you devs already know this issue, but the last Android build released is v1.4.2-417-gb2057256b from almost a month ago.

/android/src/org/ppsspp/ppsspp/NativeActivity.java:1203: error: strings in switch are not supported in -source 1.5
            switch (params) {
                   ^
    (use -source 7 or higher to enable strings in switch)

I'll see if I can reconfigure the buildbot..

--- a/android/src/org/ppsspp/ppsspp/NativeActivity.java
+++ b/android/src/org/ppsspp/ppsspp/NativeActivity.java
@@ -1200,26 +1200,20 @@ public abstract class NativeActivity extends Activity implements SurfaceHolder.C
 				NativeApp.sendMessage("permission_granted", "storage");
 			}
 		} else if (command.equals("gps_command")) {
-			switch (params) {
-				case "open":
-					if(!askForPermissions(permissionsForLocation, REQUEST_CODE_LOCATION_PERMISSION)) {
-						mLocationHelper.startLocationUpdates();
-					}
-					break;
-				case "close":
-					mLocationHelper.stopLocationUpdates();
-					break;
+			if (params.equals("open")) {
+				if (!askForPermissions(permissionsForLocation, REQUEST_CODE_LOCATION_PERMISSION)) {
+					mLocationHelper.startLocationUpdates();
+				}
+			} else if (params.equals("close")) {
+				mLocationHelper.stopLocationUpdates();
 			}
 		} else if (command.equals("camera_command")) {
-			switch (params) {
-				case "startVideo":
-					if(!askForPermissions(permissionsForCamera, REQUEST_CODE_CAMERA_PERMISSION)) {
-						mCameraHelper.startCamera();
-					}
-					break;
-				case "stopVideo":
-					mCameraHelper.stopCamera();
-					break;
+			if (params.equals("startVideo")) {
+				if (!askForPermissions(permissionsForCamera, REQUEST_CODE_CAMERA_PERMISSION)) {
+					mCameraHelper.startCamera();
+				}
+			} else if (params.equals("stopVideo")) {
+				mCameraHelper.stopCamera();
 			}
 		}
     	return false;

